tensorboard/components/vz_line_chart2/microbenchmark/README.md

```diff
@@ -0,0 +1,13 @@
+# Benchmark for vz_line_chart2
+
+To run the benchmark, do:
+
+- run `bazel run tensorboard/components/vz_line_chart2/microbenchmark:binary`
+- open browser and go to localhost:6006/benchmark.html
+- make sure the browser is in foreground as requestAnimationFrame can behave differently when tab is in background.
+- do not interact with browser that can inject noises (resize will cause layout and compositing)
+
+To add a new benchmark, do:
+
+- create a file with suffix "\_spec.ts" for consistency
+- call a `benchmark` method on './spec.js'.
```

Comment:
Nice documentation. However, it's useful (for debugging and comparison purposes) to have a set of example results here. Do you think it make sense to run the benchmark yourself on a reasonably representative machine and copy the results into this markdown file? Of course, if you do that, you want to include specs of the machine, browser version, etc.